pages/linux/sattach.md

```diff
@@ -0,0 +1,16 @@
+# sattach
+
+> Attach to a Slurm job step.
+> More information: <https://slurm.schedmd.com/sattach.html>.
+
+- Make available the IO streams (`stdout`, `stderr`, and `stdin`) of a Slurm job step:
+
+`sattach {{jobid}}.{{stepid}}`
+
+- Transmit `stdin` to the specified task only:
+
+`sattach --input-filter {{task_number}}`
```

Comment:
What do you mean by `task number`? Is this a job number, or something else?

Try to use the official language from the project itself.

Further, what do you mean by transmit to the specified task? I'm rather confused. Is this attaching interactively, or pointing something else at a job's stdin?

Reply:
"Task" is an official term. I cannot find a document providing an exact definition, but the term is mentioned several times in the docs, e.g. here https://slurm.schedmd.com/cpu_management.html#Section2

See also: https://stackoverflow.com/questions/39186698/what-does-the-ntasks-or-n-tasks-does-in-slurm

Reply:
Hmm okay, that makes sense. Thanks for the clarification.

Could you explain what you mean by `transmit to the specified task` though please?

Reply:
Basically that I send input from the current console to the specified task.